### ISSUE TITLE
Sprinkler Plugin not working

### ISSUE THREAD

**Issue by rmooreau** [jaredballou/insurgency-sourcemod#15]
Hi Jared,

I've added the Sprinkler plugin and set sm_sprinklers_enabled 1 in server.cfg but if I need at checkpoint A in ministry, the sprinklers still go off.

ozrich

**Comment by jaredballou** [comment]
Yep, this is still in progress. I remove the entities but I think there is something more complex I have to do so that it works. Noted, and working on it.
